### ISSUE TITLE
multiselectable should be two words

### ISSUE THREAD

**Issue by MichielBijl** [w3c/aria#222]
> to convey that the tablist is an accordion or a multiselectable tablist

Source: [APG: 2.2 Accordion](http://w3c.github.io/aria/practices/aria-practices.html#accordion)

It is not a reference to the attribute (`aria-multiselectable`); should be two words.

**Comment by MichielBijl** [comment]
Fixed in commit https://github.com/w3c/aria/commit/245945b7596d9b5fbda42e5d3a51733255d85ae4.
